src/App.vue

```diff
@@ -291,6 +291,33 @@ export default class App extends Vue {
       this.toggleRightPanel(null);
     }
   }
+
+  async goToNewDataset() {
+    try {
+      // Ensure we have a valid user and location before proceeding
+      if (!this.store.girderUser || !this.store.isLoggedIn) {
+        throw new Error("User must be logged in");
```

Comment:
This is unnecessary since the button will be disabled if `!store.isLoggedIn`  
You may want to change this condition to add `!this.store.girderUser` line 23